spec/user_base64_spec.rb

```diff
@@ -600,13 +610,13 @@
         context 'when a single picture is passed' do
           context 'when only data is specified' do
             it 'attaches a picture' do
-              user = User.create(pictures: base64_data)
+              user = User.create!(pictures: base64_data)
```

Comment:
could be something like:
```
let!(:user) { User.create!(pictures: picture) }
```

and then define `picture` within each spec:

```
let!(:picture) { base64_data }
```

Reply:
Agree, since it's a refactor of something already existent I'd leave it as a follow up PR